src/main/kotlin/automaton/constructor/utils/I18N.kt

```diff
@@ -0,0 +1,17 @@
+package automaton.constructor.utils
+
+import java.util.*
+
+object I18N {
+    /**
+     * List of supported Locales
+     */
+    private val locales = listOf<Locale>(Locale.ENGLISH)
+
+    val labels: ResourceBundle = ResourceBundle.getBundle("messages", getDefaultLocale())
+
+    private fun getDefaultLocale(): Locale {
+        val systemDefault = Locale(System.getProperty("user.language"))
+        return if (locales.contains(systemDefault)) systemDefault else Locale.ENGLISH
+    }
```

Comment:
Имя `labels`, на мой взгляд, неудачное, особенно если использовать static import-ы.
Ещё стоит перенести всё из `messages_en.properties` в `messages.properties`, а `messages_en.properties` оставить пустым (подробнее [здесь](https://stackoverflow.com/a/36012203)).
```suggestion
   val messages: ResourceBundle = ResourceBundle.getBundle("messages", Locale.getDefault())
```